libs/core/src/lib/select/select.component.ts

```diff
@@ -185,7 +190,12 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnInit, OnD
     }
 
     ngOnInit() {
-        // console.log(this.loading);
+        if (this.rtlService) {
+            this.rtlService.rtl
```

Comment:
Unsubscribe missing and can be simplified to observable

Reply:
observable used instead